docs/user-guide/configuring/configuration-files.md

```diff
@@ -39,6 +39,32 @@ The second way to use configuration files is to save the file wherever you would
 
 If you are using one configuration file and want ESLint to ignore any `.eslintrc.*` files, make sure to use [`--no-eslintrc`](https://eslint.org/docs/user-guide/command-line-interface#--no-eslintrc) along with the [`-c`](https://eslint.org/docs/user-guide/command-line-interface#-c---config) flag.
 
+Example configuration file:
+
+```js
```

Comment:
Did you mean this to be JSON?

```suggestion
```json
```

Reply:
Not really, the other blocks are `js` (so that comments can be used) but here it doesn't matter.